Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -874,8 +937,14 @@ def _readSingleInstanceElement(self, instanceElement, makeGlyphs=True, makeKerni
             self.readGlyphElement(glyphElement, instanceObject)
         for infoElement in instanceElement.findall("info"):
             self.readInfoElement(infoElement, instanceObject)
+        for libElement in instanceElement.findall('lib'):
+            self.readLibElement(libElement, instanceObject)
         self.documentObject.instances.append(instanceObject)
 
+    def readLibElement(self, libElement, instanceObject):
+        """Read the lib element for the given instance."""
+        instanceObject.lib = from_plist(libElement.getchildren()[0])
```

Comment:
`getchildren` is deprecated since 3.2
https://docs.python.org/3.6/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.getchildren

Reply:
you can simply get the subelements as if element was a list, without the extra call to deprecated getchildren (here and elsewhere in the code)